### PR TITLE
Initalize a creation a validator sql

### PR DESCRIPTION
**Pull request by Matheus-Oliveira1**
Start a create a validator for sql.

### CODE REVIEW

File: src/lib/formatters/json.py [hunk 1]
```diff
@@ -33,12 +33,3 @@ def format(self, dataframe: DataFrame, path_or_buffer) -> None:
...
                 path_or_buf=path_or_buffer, force_ascii=False, **parameters)
-
-    @staticmethod
-    def check(options):
-        indent = options.get("indent")
-        if indent in ["pretty", "minify", "", None]:
-            pass
-        else:
-            raise ValueError(
-                            "json-array suports only 'pretty' and 'minify'")
```
Comment (tadeuif) [inline src/lib/formatters/json.py:44]: Why it was excluded? Just to know :)

Comment (lucasfernando) [reply src/lib/formatters/json.py:44]: this method is not being called!

File: src/lib/formatters/sql.py [hunk 2]
```diff
@@ -17,15 +17,39 @@ def __init__(self, specification):
...
     def rules():
+        def replace_rule(options):
+            schema = options.get("schema")
+            for key in schema.keys():
+                field = schema.get(key)
+                mode = options.get("mode")
+                if mode == "replace":
+                    if not isinstance(field.get("sqltype", None), str):
+                        raise ValueError(
+                            "The Mode replace needs'sqltype' in Schema fields")
```
Comment (lucasfernando) [inline src/lib/formatters/sql.py]: ta faltando um espaço entre o needs e o 'sqltype'

**Review by lucasfernando — APPROVED** [review]
LGTM

**Pull request merged**